data/frame.go

```diff
@@ -56,6 +56,13 @@ func (f *Frame) AppendRow(vals ...interface{}) {
 	}
 }
 
+// InsertAt adds values in the field vector (identyfied by fieldIdx) at the index specified by vectorIdx
+func (f *Frame) InsertRowAt(fieldIdx int, vals []interface{}) {
```

Comment:
Generally we do `vals ...interface{})`, this way, if you want want to, you cant pass `InsertRowAt(2, "foo", 2, 2.2)` etc, (nice when testing).

In particular because you can't `[]interface{1, "foo", 2.2}` in go.